.gitmodules

```diff
@@ -329,3 +329,15 @@
 	path = third_party/infineon/psoc6/psoc6_sdk/libs/lwip-network-interface-integration
 	url = https://github.com/Infineon/lwip-network-interface-integration.git
 	platforms = infineon
+[submodule "third_party/abseil-cpp/src"]
```

Comment:
This is checking out these submodules for all platform targets.  Does that make sense?